pkg/k8s/apis/cilium.io/v2/types.go

```diff
@@ -458,6 +458,41 @@ type EndpointIdentity struct {
 	LabelsSHA256 string `json:"labelsSHA256,omitempty"`
 }
 
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// CiliumIdentity is an identity managed by Cilium
+type CiliumIdentity struct {
+	// +k8s:openapi-gen=false
+	metav1.TypeMeta `json:",inline"`
+	// +k8s:openapi-gen=false
+	metav1.ObjectMeta `json:"metadata"`
+
+	// SecurityLabels is the source-of-truth set of labels for this identity. The
+	// labels placed in metav1.ObjectMeta are there for convenience when
```

Comment:
What exactly does "for convenience" mean in this case? Elaborate on whether the labels in `metav1.ObjectMeta` may be different than `SecurityLabels`.